src/bundle/Resources/public/js/scripts/helpers/middle.ellipsis.js

```diff
@@ -0,0 +1,16 @@
+(function (global, doc, ibexa) {
+    const parseAll = () => {
+        const middleEllipsisContainers = [...doc.querySelectorAll('.ibexa-middle-ellipsis')];
+
+        middleEllipsisContainers.forEach((middleEllipsisContainer) => {
+            const partStart = middleEllipsisContainer.querySelector('.ibexa-middle-ellipsis__name--start');
+
+            middleEllipsisContainer.classList.toggle('ibexa-middle-ellipsis--ellipsized', partStart.scrollWidth > partStart.offsetWidth);
+            ibexa.helpers.tooltips.parse(middleEllipsisContainer);
+        });
+    };
+
+    ibexa.addConfig('helpers.middleEllipsis', {
```

Comment:
I would split it just in case in the future there will be other types:
```suggestion
    ibexa.addConfig('helpers.ellipsis.middle', {
```